### ISSUE TITLE
Add ability for user to add and remove categories from each topic (i.e. income, or giving).

### ISSUE THREAD

**Issue by wsurles** [wsurles/finance_tracker#1]

**Comment by mrsoltys** [comment]
I agree this is a much needed feature!  Court has custom income categories for some of her more common clients, and because these aren't recognized by the "income" tab, they automatically go to spending where they're not appropriate.  

Similarly, we pay estimated taxes, and I think this should be grouped in with Income and not with Spending. But the user should make these calls
